### PR TITLE
Update glslang, add x-checker-data, update runtime

### LINKED ISSUE #9: Update Snes9x to 19.08

### PR DESCRIPTION
**Pull request by lionirdeadman**
fixes #9

### CODE REVIEW

**Comment by lionirdeadman** [comment]
bot, build

**Comment by lionirdeadman** [comment]
Why does it fail to clone?.. Works locally..

**Comment by lionirdeadman** [comment]
bot, build

File: com.snes9x.Snes9x.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
         "--socket=x11",
         "--socket=wayland",
         "--socket=pulseaudio",
-        "--filesystem=home"
+        "--filesystem=home:ro"
```
Comment (TingPing) [inline com.snes9x.Snes9x.json:14]: Lets keep unrelated changes separate.

**Comment by TingPing** [comment]
Regarding the git failure, maybe just modify it so snes9x stops cloning these submodules, we already build `glslang` so the bundled one isn't used (I'd imagine).

**Comment by lionirdeadman** [comment]
The slang option doesn't stop the submodule cloning so I'm not sure how to do that.

**Comment by TingPing** [comment]
`disable-submodules` can be set in the manifest.

**Comment by lionirdeadman** [comment]
Close in favour of #11
